src/SpriteComponent.cpp

```diff
@@ -1,10 +1,11 @@
 #include "SpriteComponent.h"
 #include "DuckyMath.h"
+#include "ResourceManager.h"
 
 
-void SpriteComponent::SetTexture(const char* path)
+void SpriteComponent::SetTexture(const char* name)
 {
-	Texture.loadFromFile(path);
+	Texture = ResourceManager::Get().GetTexture(name);
```

Comment:
Don't need it. SpriteComponent should only have sf::Sprite member and then
`Sprite.setTexture(ResourceManager::Get().GetTexture(name));`